src/cmd/library_add.js

```diff
@@ -51,7 +51,7 @@ export class LibraryAddCommand {
 		await this.loadProject();
 		const library = await this.fetchLibrary(lib.name, lib.version);
 		await this.addLibraryToProject(library);
-		await this.saveProject();
+		return this.saveProject();
```

Comment:
before the whenjs removal we used `pipeline`. According with the documentation this method always returns the result of the last task, in order to keep that behavior we need to return this.